core/cosmos-fonts/index.js

```diff
@@ -0,0 +1,61 @@
+import { fonts, misc } from '@auth0/cosmos-tokens'
+
+const insertAtTheStart = styles => {
+  let tag = document.getElementById('cosmos-fonts')
```

Comment:
In the server side we'd get an `undefined has not property 'getElementById'` kind error because `document` is not a thing there.

Reply:
should we just add a typeof undefined check?

Reply:
that would do it, yeah. we would still need to find a way to apply global styles when cosmos is rendered in the server (maybe with `injectGlobal`?)